tidb-cloud/third-party-integrations.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Would you please add "monitoring" to the filename? Thanks

Reply:
Done